CHANGELOG.md

```diff
@@ -70,6 +70,42 @@ $govuk-suppressed-warnings: (
 
 This was added in [#2911 Add warning suppression functionality](https://github.com/alphagov/govuk-frontend/pull/2911)
 
+#### Configure components in JavaScript
+
+JavaScript components receiving configuration through data attributes now accept the same configuration options when instantiated in JavaScript. These components are:
+
+- the `Button` component, for its `preventDoubleClick` option (matching `data-prevent-double-click`)
+- the `CharacterCount` component, for its `maxlength`,`maxwords` and `threshold` options (matching `data-maxlength`, `data-maxwords` and `data-threshold`, respectively)
+- the `ErrorSummary` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+- the `NotificationBanner` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+
+You can omit these options when calling the Nunjucks macro and provide them:
+
+- at component instantiation, in a configuration object as second argument, for example:
+
+```js
+var errorSummary = document.querySelector('[data-module="error-summary"]')
+new GOVUKFrontend.ErrorSummary(errorSummary, {disableAutoFocus: true})
+```
```

Comment:
Suggest indenting this so that it falls within the list item:

```suggestion
  ```js
  var errorSummary = document.querySelector('[data-module="error-summary"]')
  new GOVUKFrontend.ErrorSummary(errorSummary, {disableAutoFocus: true})
  ```
```

Reply:
`disableAutoFocus` is going to be irrelevant for most if not all service teams – we only really have it for use in examples in the Design System.

It might be worth providing examples for Button with the `preventDoubleClick` option instead?